### PR TITLE
docs(configuration): added BigQuery CUBEJS_DB_BQ_LOCATION env in documentation

### PR DESCRIPTION
**Pull request by HJ29**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Issue Reference this PR resolves**

No issue opened

**Description of Changes Made (if issue reference is not provided)**

if bigquery db location isn't default, cubejs query will fail when query job with the following error
Error: Error: Not found: Job [project_id]:[job_id]
for my case, my bigquery db location is asia-southeast1
the correct job id should be [project_id]:asia-southeast1:[job_id]

I spend some time to find a way to configure bigquery location,
end up there is a pr here #2104 to set bigquery location in env, but without any documentation

### CODE REVIEW

**Comment by rpaik** [comment]
@hassankhan can you help with the review? 🙏

**Review by igorlukanin — CHANGES_REQUESTED** [review]
Thanks for the PR, @HJ29!

I think it would also make sense to add a note to the [BigQuery-specific section](https://cube.dev/docs/connecting-to-the-database#notes-google-big-query) below on the same page, and also add a row to the table on the [configuration page](https://cube.dev/docs/reference/environment-variables#database-connection).

File: docs/content/Configuration/Connecting-to-the-Database.md [hunk 1]
```diff
@@ -192,6 +192,14 @@ CUBEJS_DB_BQ_CREDENTIALS=$(cat /path/to/key-file.json | base64)
 You can learn more about acquiring Google BigQuery credentials
 [here][link-bigquery-getting-started] and [here][link-bigquery-credentials].
 
+You can set the dataset location `CUBEJS_DB_BQ_LOCATION` environment variable.
```
Comment (hassankhan) [inline docs/content/Configuration/Connecting-to-the-Database.md:195]:
```suggestion
You can set the dataset location using the `CUBEJS_DB_BQ_LOCATION` environment variable.
```

**Review by hassankhan — APPROVED** [review]
Awesome work @HJ29 🙌  Thank you so much for your contribution 😀

**Pull request merged**